react/components/ProductSummaryInline.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import { path } from 'ramda'
+import classNames from 'classnames'
+import { Link } from 'vtex.render-runtime'
+
+import AttachmentList from './AttachmentList'
+import ImageLoader from './ImageLoader'
+import ProductImage from './ProductImage'
+import ProductSummaryBuyButton from './ProductSummaryBuyButton'
+import ProductQuantityStepper from './ProductQuantityStepper'
+import ProductSummaryPrice from './ProductSummaryPrice'
+import ProductSummaryName from './ProductSummaryName'
+
+import productSummary from '../productSummary.css'
+
+class ProductSummaryInline extends Component {
+  render() {
+    const {
+      product,
+      showBorders,
+      handleMouseEnter,
+      handleMouseLeave,
+      handleItemsStateUpdate,
+      actionOnClick,
+      imageProps,
+      nameProps,
+      priceProps,
+      buyButtonProps,
+    } = this.props
+
+    const classes = classNames(
+      productSummary.container,
+      productSummary.containerInline,
+      'overflow-hidden br3 h-100 w-100'
+    )
+
+    const elementClasses = classNames(`${productSummary.element} pointer ph2 pt3 pb4 flex flex-column`, {
+      'bb b--muted-4 mh2 mt2': showBorders,
+    })
+
+    const nameClasses = {
+      containerClass: 'flex items-start justify-left w-100 pt5 t-mini pb2',
+      brandNameClass: 't-body c-on-base',
+    }
+
+    const sellingPrice = path(['sku', 'seller', 'commertialOffer', 'Price'], product)
+    const priceClasses = {
+      containerClass: classNames('flex flex-column', {
+        [`${productSummary.priceContainer} pv5`]: !showBorders,
+      }),
+      sellingPriceClass: classNames('dib ph2 t-body t-heading-5-ns', {
+        't-small t-body-ns': sellingPrice > 1000,
```

Comment:
These classes are not being applied when the price of a product is more than 1000. Also, its gonna have two typography classes when this condition is fulfilled: `t-body` and `t-small`